rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -326,12 +331,25 @@ def __init__(
         self._label_data: Optional[RasaModelData] = None
         self._data_example: Optional[Dict[Text, List[np.ndarray]]] = None
 
+        split_entities_config = self.component_config.get(
+            SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE
+        )
+        if isinstance(split_entities_config, bool):
+            split_entities_config = {SPLIT_ENTITIES_BY_COMMA: split_entities_config}
+        else:
+            split_entities_config[SPLIT_ENTITIES_BY_COMMA] = self.defaults[
+                SPLIT_ENTITIES_BY_COMMA
+            ]
+        self.split_entities_config = split_entities_config
```

Comment:
I would move this to a new method in the `EntityExtractor`. As both, `DIETClassifier` and the `CRFEntityExtractor` are subclasses of the `EntityExtractor` they can call the new method in the init.